scripts/env-data.sh

```diff
@@ -3,6 +3,7 @@ POSTGRES_MAJOR_VERSION=$(cat /tmp/pg_version.txt)
 POSTGIS_MAJOR=$(cat /tmp/pg_major_version.txt)
 POSTGIS_MINOR_RELEASE=$(cat /tmp/pg_minor_version.txt)
 DEFAULT_DATADIR="/var/lib/postgresql/${POSTGRES_MAJOR_VERSION}/main"
+POSTGRES_INITDB_WALDIR="/opt/postgresql/${POSTGRES_MAJOR_VERSION}/pg_waldir"
```

Comment:
So, to reconfirm, `POSTGRES_INITDB_WALDIR` uses `/opt/postgresql` directory while `DEFAULT_DATADIR` uses `/var/lib/postgresql` ?

Reply:
Yes, I can change the location from /opt/postgresql to another location if you feel so. The idea is to be able to mount the two in different folders as done in  Postgres official images